doc/pages/architecture/020-components.md

```diff
@@ -4,12 +4,24 @@ Astarte is a distributed system interacting over AMQP, as explained in [Design P
 
 ## Pairing
 
+Pairing takes care of Device Authentication and Authorization. It interacts with Astarte's CA and orchestrates the way devices connect and interact with Transports. It also allows Agents to perform Device Registration. Agent, Device and Pairing interaction is described in detail [here](050-pairing_mechanism.html).
+
 ## Data Updater Plant (DUP)
 
+Data Updater Plant is a replicable, scalable component which takes care of the ingestion pipeline. It gathers data from devices and orchestrates data flow amongst other components. It is, arguably, the most critical component of the system and the most resource hungry - the way DUP is deployed, replicated and configured has a tremendous impact on Astarte's performances, especially when dealing with massive data flows.
```

Comment:
it is not replicable yet. Should we remove this?

Reply:
Nope. From a design POV it is, and it's all that matters as here we're merely describing how the system looks like. So indeed it belongs there.